package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stripe",
-  "version": "6.31.2",
+  "version": "7.0.0",
```

Comment:
No need to change that, the release script will handle it.

Reply:
Can I change it anyway? makes intent more clear

Reply:
Sure, it might just be annoying if the v7 branch is long-lived and we release more 6.x versions since it will cause merge conflicts.